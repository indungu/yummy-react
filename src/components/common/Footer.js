import React from 'react';

const Footer = () => {
  return (
    <footer className="footer">
      <div className="container">
        <span className="text-white text-center">
          Designed with <i className="fa fa-heart text-warning"></i> by <a href="https://github.com/indungu">indungu</a>
        </span>
      </div>
    </footer>
  );
};

export default Footer;
