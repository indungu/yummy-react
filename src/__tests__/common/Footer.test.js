import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Footer from '../../components/common/Footer';

describe('Login', () => {
  const wrapper = shallow(<Footer />);
  it('Renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
