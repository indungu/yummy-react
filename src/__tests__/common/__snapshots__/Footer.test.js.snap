// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Renders properly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span
      className="text-white text-center"
    >
      Designed with 
      <i
        className="fa fa-heart text-warning"
      />
       by 
      <a
        href="https://github.com/indungu"
      >
        indungu
      </a>
    </span>
  </div>
</footer>
`;
